packages/web3-eth-ens/src/ens.ts

```diff
@@ -0,0 +1,167 @@
+import { Address } from 'web3-utils';
+import { RevertInstructionError } from 'web3-common';
+import { NonPayableCallOptions, TransactionReceipt } from 'web3-eth-contract';
+import { Registry } from './registry';
+
+export class ENS {
+	private _registryAddress: string;
+	private readonly registry: Registry;
+
+	public constructor() {
+		this.registry = new Registry();
+		this._registryAddress = ''; // TODO change this when eth.net is finished
```

Comment:
you should also have a [consts](https://github.com/ChainSafe/web3.js/blob/bd6a9d86bbb9dc6e1fbe74216f9d7d44d7ba2b47/packages/web3-eth-ens/src/config.js#L28) file for registry address and other const values , and you should use that to set registry addr

Reply:
and should pass that to `new Registry()` constructor to registry contract instance creation